### ISSUE TITLE
Flathub release doesn't work on Ubuntu 20.04

### ISSUE THREAD

**Issue by despens** [mrvladus/Errands#319]
Errands 46.2.3 installed via Flatpak starts up but never gets to show the main window. Python errors below:

<details>

<summary>Error log</summary>

```console
$ flatpak run io.github.mrvladus.List

-------------------------------------------------
Starting Errands 46.2.3 at "2024 26 Juni 08:16:54"
-------------------------------------------------

[DEBUG] Application: Startup
[DEBUG] Data: Initialize
[DEBUG] Data: Read data
[DEBUG] GSettings: Initialize
[DEBUG] Application: Checking autostart
[DEBUG] Main Window: Load
[DEBUG] Today Page: Load
[DEBUG] Today Page: Update UI
[DEBUG] Trash Page: Load
[DEBUG] Sidebar: Load Task Lists
[DEBUG] Sidebar: Add Task List 'defaultcalendar'
[INFO] Task List defaultcalendar: Load Tasks
[DEBUG] Data: Write data
[DEBUG] Task List 'defaultcalendar': Update title
[DEBUG] Task List Row: Update UI 'defaultcalendar'
[DEBUG] Sidebar: Add Task List 'schulferien'
[INFO] Task List schulferien: Load Tasks
[DEBUG] Data: Write data
[DEBUG] Task List 'schulferien': Update title
[DEBUG] Task List Row: Update UI 'schulferien'
[DEBUG] Sidebar: Select last opened page
Traceback (most recent call last):
  File "/app/share/errands/errands/application.py", line 157, in do_startup
    State.main_window = Window(application=State.application)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/widgets/window.py", line 41, in __init__
    self.__finish_load()
  File "/app/share/errands/errands/widgets/window.py", line 140, in __finish_load
    State.trash_sidebar_row.update_ui()
  File "/app/share/errands/errands/widgets/trash/trash_sidebar_row.py", line 94, in update_ui
    State.trash_page.update_ui()
  File "/app/share/errands/errands/widgets/trash/trash.py", line 136, in update_ui
    item.update_ui()
  File "/app/share/errands/errands/widgets/trash/trash.py", line 199, in update_ui
    self.set_subtitle(UserData.get_list_prop(self.task_dict.list_uid, "name"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/lib/data.py", line 345, in get_list_prop
    return getattr(self.get_list(list_uid), prop)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'name'
[DEBUG] Application: Activate
[DEBUG] Main Window: Load
[DEBUG] Today Page: Load
[DEBUG] Today Page: Update UI
[DEBUG] Trash Page: Load
[DEBUG] Sidebar: Load Task Lists
[DEBUG] Sidebar: Add Task List 'defaultcalendar'
[INFO] Task List defaultcalendar: Load Tasks
[DEBUG] Data: Write data
[DEBUG] Task List 'defaultcalendar': Update title
[DEBUG] Task List Row: Update UI 'defaultcalendar'
[DEBUG] Sidebar: Add Task List 'schulferien'
[INFO] Task List schulferien: Load Tasks
[DEBUG] Data: Write data
[DEBUG] Task List 'schulferien': Update title
[DEBUG] Task List Row: Update UI 'schulferien'
[DEBUG] Sidebar: Select last opened page
Traceback (most recent call last):
  File "/app/share/errands/errands/application.py", line 165, in do_activate
    State.main_window = Window(application=State.application)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/widgets/window.py", line 41, in __init__
    self.__finish_load()
  File "/app/share/errands/errands/widgets/window.py", line 140, in __finish_load
    State.trash_sidebar_row.update_ui()
  File "/app/share/errands/errands/widgets/trash/trash_sidebar_row.py", line 94, in update_ui
    State.trash_page.update_ui()
  File "/app/share/errands/errands/widgets/trash/trash.py", line 136, in update_ui
    item.update_ui()
  File "/app/share/errands/errands/widgets/trash/trash.py", line 199, in update_ui
    self.set_subtitle(UserData.get_list_prop(self.task_dict.list_uid, "name"))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/lib/data.py", line 345, in get_list_prop
    return getattr(self.get_list(list_uid), prop)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'name'
^C
```

</details>

**Comment by mrvladus** [comment]
Try this:
- Go to `$HOME/.var/app/io.github.mrvladus.List/data/errands`
- Copy `data.json` file (and `data.old.json` if it's there) somewhere like `Downloads`
- Delete `$HOME/.var/app/io.github.mrvladus.List` directory
- Launch and close Errands
- Copy peviosly copied `data.json` to `$HOME/.var/app/io.github.mrvladus.List/data/errands` 
- Launch Errands again

**Comment by despens** [comment]
Thanks for your quick reaction @mrvladus!

I followed the steps you outlined, the exact same error messages appears on the console after I restored my `data.json` file.

Maybe I should have mentioned that I am syncing tasks with my nextcloud server. Running the exact same version of Errands on another computer works without any problems.

I could try copying the data over from that other computer, but would like to help identifying the issue.
